app/views/movies/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= content_tag :h1, "Discover Your Next Watch"  %>
+<%= button_to "Find Top Rated Movies", "/movies", method: :get %>
+
+<%= form_tag "/movies", method: :get %>
+  <%= search_field_tag(:search, params[:search]) %>
+<%= submit_tag ("Search By Title") %>
+
+
+  <% @movies.each do |movie| %>
+    <div class="movie">
+      <div class="title">
+       <%= link_to "#{movie[:title]}" %>
```

Comment:
We might not need to string interpolate here, but the path we add (for that movie's show page) will have quotes. For example:
-`<%=link_to item.name, "/items/#{item.id}" %>`

Reply:
I think you're right, adjusting that!